src/fiatExchanges/ReviewScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import { RouteProp } from '@react-navigation/native'
+import { StackScreenProps } from '@react-navigation/stack'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, SafeAreaView, StyleSheet, Text, View } from 'react-native'
+import BackButton from 'src/components/BackButton'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import LineItemRow from 'src/components/LineItemRow'
+import i18n from 'src/i18n'
+import { emptyHeader } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { CICOFlow } from './utils'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectReview>
+
+export default function FiatConnectReviewScreen({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  // TODO(satish): get all hardcoded fields from props
+  const { flow } = route.params
+
+  return (
+    <SafeAreaView style={styles.content}>
+      <View>
```

Comment:
Nit: is this view necessary?

Also sidenote: for any View that doesn't have any props you can just use `<>...</>` instead

Reply:
yes, this was necessary because otherwise all elements on the screen were justified. This ensures only the top button at the bottom is justified.